shardingsphere-infra/shardingsphere-infra-binder/src/main/java/org/apache/shardingsphere/infra/binder/segment/select/projection/ProjectionsContext.java

```diff
@@ -38,18 +38,18 @@
 @Getter
 @ToString
 public final class ProjectionsContext {
-    
+
```

Comment:
There is no change on the class ProjectionsContext.java beside the indent, could you revert the change of this class?